docs/how-to/create-a-router/configure-traffic-rules.md

```diff
@@ -8,15 +8,31 @@ This step is **optional** and the default behaviour will not discriminate betwee
 
 It is also possible to configure your router such that each request is only dispatched to a subset of the configured routes, based on some request specific conditions. For example, you might have some models, trained on geography-specific data. So, in this scenario, you want to call `mock` if the request contains header `"X-Region: region-A"` or `model-b` if `X-Region` header equals `"region-B"`. It's possible to achieve this by configuring traffic rules on your router.
 
-Traffic rules define which routes should be "activated" for a particular request to your router. Each rule is defined by one or more request conditions and one or more routes that would be activated if the request satisfies the conditions of the rule. Routes that are not part of any traffic rules will be called for each request.
+Traffic rules define which routes should be "activated" for a particular request to your router. Each rule is defined by one or more request conditions and one or more routes that would be activated if the request satisfies the conditions of the rule. 
 
-Rules are matched against the incoming request in the order in which they are defined. This property can be used to create one or more specific rules over a general rule. Consider the following example, where a router has rules defined such that requests are routed by the country of origin (ID, SG, etc.). In addition, if the routing logic must be altered for a certain service type in a country, this rule (the 'specific' rule) can be defined before the general rule.
+When traffic rules are configured, the following behaviour is expected:
+- A Default traffic rule is required, which has no request conditions and act as a fallback rule for requests that don't match any of the configured traffic rule conditions. At least 1 route should be configured for the Default traffic rule.
+- All routes should be used by at least 1 traffic rule, i.e no dangling routes.
+- When a Nop or Standard Ensembler is configured (See: [Configure Ensembler](./configure-ensembler.md)), the `default_route_id` must be set and present in every traffic rule.
```

Comment:
> the `default_route_id` must be set and present in every traffic rule

Can we phrase this as "the final / fallback route must be present in every traffic rule, including the default rule" (since `default_route_id` is an internal detail) ?